tutorials/tour-of-restate-rust/src/main.rs

```diff
@@ -0,0 +1,27 @@
+mod cart_object;
+mod checkout_service;
+mod ticket_object;
+
+pub mod auxiliary;
+
+use restate_sdk::prelude::*;
+
+#[tokio::main]
+async fn main() {
+    // To enable logging
+    tracing_subscriber::fmt::init();
+
+    HttpServer::new(
+        Endpoint::builder()
+            .bind(cart_object::CartObject::serve(cart_object::CartObjectImpl))
```

Comment:
you should be able to simply to

```suggestion
            .bind(cart_object::CartObjectImpl.serve())
```

But you need to import the `CartObject` trait